src/app/jscaip/tests/RulesUtils.spec.ts

```diff
@@ -96,14 +96,9 @@ export class RulesUtils {
             minimax.getBoardValue(new MGPNode(weakerState, MGPOptional.empty(), weakMove)).value;
         const strongValue: number =
             minimax.getBoardValue(new MGPNode(strongerState, MGPOptional.empty(), strongMove)).value;
-        expect(weakerState.getCurrentPlayer())
-            .withContext('expectSecondStateToBeBetterThanFirst expects states for the same player')
-            .toBe(strongerState.getCurrentPlayer());
-        if (weakerState.getCurrentPlayer() === Player.ZERO) {
-            expect(weakValue).toBeGreaterThan(strongValue);
-        } else {
-            expect(weakValue).toBeLessThan(strongValue);
-        }
+        const multipliedWeakValue: number = weakValue * weakerState.getCurrentPlayer().getScoreModifier();
+        const multipliedStrongValue: number = strongValue * strongerState.getCurrentPlayer().getScoreModifier();
+        expect(multipliedWeakValue).toBeLessThan(multipliedStrongValue);
```

Comment:
Soucis supplémentaire à ce bug là, une victoire poserais problème, *-1 l'un des MAX_VALUE/MIN_VALUE (j'sais plus lequel) ne marche pas trop, si je n'm'abuse !

Math.floor(Number.MIN_VALUE * -1) => -1 :D

Reply:
yep, corrigé avec le +for